sdk/developer-signing/Azure.Developer.Signing/tests/Samples/Sample_CreateCertificateProfileClient.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core.TestFramework;
+
+namespace Azure.Developer.Signing.Tests.Samples
+{
+    public partial class DeveloperSigningSample : SamplesBase<DevSigningClientTestEnvironment>
+    {
+        public void CreateClients(string region)
```

Comment:
nit:  did we intend to mark this with `[Test]`?

Reply:
Yes, we should, thanks.